ui/src/testing/factories/state.ts

```diff
@@ -6,24 +6,32 @@ import { user } from "./user";
 import type { AuthState, UserState } from "app/store/user/types";
 import type { MessageState } from "app/store/message/types";
 import type { NotificationState } from "app/store/notification/types";
+import type { PodState } from "app/store/pod/types";
 
-export const authState = define<AuthState>({
-  errors: null,
-  loaded: true,
+const defaultState = {
+  errors: () => ({}),
+  loaded: false,
   loading: false,
-  saved: true,
+  saved: false,
   saving: false,
+};
+
+export const authState = define<AuthState>({
+  ...defaultState,
   user,
 });
 
 export const userState = define<UserState>({
+  ...defaultState,
   auth: authState,
-  errors: null,
   items: array(user),
-  loaded: true,
-  loading: false,
-  saved: true,
-  saving: false,
+});
+
+export const podState = define<PodState>({
+  ...defaultState,
+  items: () => [],
```

Comment:
You could set this on defaultState and have the factories (e.g. userState) override that property.